stingray/deadtime/fad.py

```diff
@@ -77,20 +48,18 @@ def calculate_FAD_correction(lc1, lc2, segment_size, norm="none", gti=None,
 
     Parameters
     ----------
-    lc1: class:`stingray.ligthtcurve.Lightcurve`
-        Light curve from channel 1
-    lc2: class:`stingray.ligthtcurve.Lightcurve`
-        Light curve from channel 2. Must be strictly simultaneous to ``lc1``
-        and have the same binning time. Also, it must be strictly independent,
-        e.g. from a different detector. There must be no dead time cross-talk
-        between the two light curves.
+    data1 : `Lightcurve` or `EventList`
+        Input data for channel 1
+    data2 : `Lightcurve` or `EventList`
+        Input data for channel 2
```

Comment:
It is not a breaking change, because the function with the old name remains, still accepting lc1 and lc2. This is a new function, that also accepts event lists :). Good point about the description